modules/sdk-core/src/coins/fiataed.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @prettier
+ */
+import {
+  BaseCoin,
+  BitGoBase,
+  KeyPair,
+  MethodNotImplementedError,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '../';
+
+export class FiatAED extends BaseCoin {
```

Comment:
Would it be easier to just create a new package for the fiat coins, `abstract-fiat`? And then each of these fiat coins can extend a common FiatBaseCoin class.